cmake/modules/RootBuildOptions.cmake

```diff
@@ -293,6 +293,10 @@ if(NOT CMAKE_CXX_STANDARD GREATER 11)
   set(webui_defvalue OFF)
 endif()
 
+if("${webui}" STREQUAL "OFF")
```

Comment:
`if (NOT webui AND NOT "${webui}" STREQUAL "")` will also catch `"0"`, `"NO"`, `"FALSE"` etc.